include/sycl_ext_complex.hpp

```diff
@@ -2093,7 +2093,7 @@ sycl::marray<T, N> exclusive_scan_over_group(Group g, sycl::marray<V, N> x,
 
 // Use sycl::detail::loop if the compiler is intel/llvm's clang
 #if defined(SYCL_IMPLEMENTATION_ONEAPI) && !defined(__INTEL_LLVM_COMPILER)
-  sycl::detail::detail_loop<N>([&](size_t s) {
+  sycl::detail::loop<N>([&](size_t s) {
```

Comment:
Do you think it will make tense to implement our `sycl::complex::detail::loop`? So we can put the `ifdef` inside, and use it everywhere. Maybe terser?
We use it only 3 time so maybe not a big deal